app/common/util/src/main/java/io/syndesis/common/util/Names.java

```diff
@@ -16,36 +16,40 @@
 package io.syndesis.common.util;
 
 import java.util.Locale;
+import static io.syndesis.common.util.Strings.truncate;
 
 public final class Names {
```

Comment:
What is actually the difference to the `Labels` class ? For me the sanitize() algorithm looks the same.